vrpn_ForceDevice.C

```diff
@@ -2742,8 +2742,7 @@ void vrpn_ForceDevice_Remote::send(const char *msgbuf, vrpn_int32 len,
         }
     }
 
-    // HP compiler won't let you delete a const argument.
-    delete[](char *)msgbuf;
+    delete[]msgbuf;
```

Comment:
I assume the HP compiler referred to in the old comment is an outdated HP-UX compiler that it's OK to drop support for? @russell-taylor